spring-xd-dirt/src/main/resources/application.yml

```diff
@@ -349,4 +349,13 @@ transport: local
 analytics: memory
 embeddedHsql: true
 
----
\ No newline at end of file
+---
+# Kryo uses references by default. This feature must be enabled unless we know the payload types being serialized do not
+# contain circular refefences. Disabling this feature improves performance if it is not needed.
```

Comment:
Instead of using `enabling` or `disabling`, can we specify what happens it this is set to `true` or `false`?  That way it will be very clear to the end user.